### ISSUE TITLE
Undefined symbol: realm::set_default_realm_file_directory

### ISSUE THREAD

**Issue by haswalt** [realm/realm-js#5633]
Cannot build version `11.6.0` with ReactNative `0.71.4` on Xcode `Version 14.2 (14C18)`.

Fails with this error:

```
Undefined symbol: realm::set_default_realm_file_directory(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >)
```

Rolling back to `11.5.2` builds correctly.

**Comment by kneth** [comment]
@haswalt Thank you for reporting. We will investigate.

**Comment by kneth** [comment]
@haswalt We have a fix in https://github.com/realm/realm-js/pull/5635, and once reviewed we can do a release

**Comment by kneth** [comment]
@haswalt A fix has been released in [v11.7.0](https://github.com/realm/realm-js/releases/tag/v11.7.0)
